winml/test/common/test.h

```diff
@@ -1,10 +1,18 @@
+#pragma once
+
 using VoidTest = void (*)();
 using SetupTest = VoidTest;
 
+constexpr bool alwaysTrue() {
+    return true;
+}
+#define WINML_SUPRESS_UNREACHABLE_BELOW(statement)    \
+    if (alwaysTrue()) { statement; }
```

Comment:
I tried if constexpr which would ensure the unused code doesn't get compiled, but VS still shows a warning (looks like a bug: https://developercommunity.visualstudio.com/content/problem/421660/warning-c4702-is-overzealous-with-if-constexpr-gua.html)

Reply:
Is it required to fix C4702 warnings ?

Reply:
Yes, the code after GPUTEST is unreachable if building without DML